.travis.yml

```diff
@@ -10,7 +11,7 @@ cache:
     - $HOME/.composer/cache
 
 install:
-  - composer install --no-interaction --dev
+  - composer install --no-interaction
```

Comment:
The `--dev` option is deprecated and it's fine to removed here.
The deprecated warning message is as follows:

```
You are using the deprecated option "dev". Dev packages are installed by default now.
```